fastlane_bot/events/utils.py

```diff
@@ -2063,28 +2007,4 @@ def check_and_approve_tokens(tokens: List, cfg) -> bool:
     :param cfg: the config object
 
     """
-
```

Comment:
That's the purpose of this task - cleanup.

Reply:
cleanup is about either removing unused code or doing the same thing more efficiently. How confident are you that the code still operates the same way in all relevant real world scenarios?

Reply:
I'm actually pretty confident that some stuff may have been broken as a result of this PR, which we shall fix by testing it, and not by keeping the previous code or reverting back to it (due to its poor shape).
The purpose of this PR is to end up with cleaner and more robust code, which will be beneficial in the long term (and most likely also in the short term).

Reply:
because of the poor state of our tests I am worried that this code may break something not-so-obvious in the bot, and I am not confident that we'll be able to catch it before it goes into production

Reply:
I agree, but the same arguments holds even prior to all these changes.
The code is already broken in a bunch of non-obvious places, which occasionally emerge during runtime.
So I've opted to demolish and rebuild this part of the project.
Ultimately, the TX-Helper functions should all be fairly simple, readable and easy to implement.
As it stands, there are a bunch of culprits, including functions which return a different number of argument on different execution paths (sometimes even not returning anything when they should be), piece of code which are duplicated (or worse, partially-duplicated) across several different modules, etc.
This needs to be resolved.